### PR TITLE
Publications and Research Line

### PR DESCRIPTION
**Pull request by lorenarm96**

### CODE REVIEW

File: themes/timer-hugo/assets/css/style.css [hunk 1]
```diff
@@ -297,7 +297,8 @@ a:hover {
   background-position: center center;
   background-size: cover;
   background-attachment: fixed;
-  padding: 230px 0;
+  padding: 150px 0;
+  padding-bottom: 50px;
```
Comment (jaimergp) [inline themes/timer-hugo/assets/css/style.css:301]: This change doesn't render well in my setup! I'll revert for now. What were you trying to fix here?

image

Comment (lorenarm96) [reply themes/timer-hugo/assets/css/style.css:301]: we were trying to reduce the space under the banner, JeanDi does not like it so big

Comment (jaimergp) [reply themes/timer-hugo/assets/css/style.css:301]: Oops, sorry, this comment didn't belong here.

File: themes/timer-hugo/layouts/partials/header.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
             <div class="col-lg-12">
                 <nav class="navbar navbar-expand-lg navbar-light bg-light">
                     <a class="navbar-brand" href="{{ .Site.BaseURL }}">
-                        <img src="{{ .Site.Params.logo | absURL }}" height="50em" width="auto" alt="logo">
+                        <img src="{{ .Site.Params.logo | absURL }}" height="100em" width="auto" alt="logo">
```
Comment (jaimergp) [inline themes/timer-hugo/layouts/partials/header.html:7]: I meant this: https://github.com/insilichem/insilichem_com/pull/6/files#r425237591

File: themes/timer-hugo/assets/css/style.css (hunk 1 above)
Comment (jaimergp) [reply themes/timer-hugo/assets/css/style.css:301]: It's fixed now.

**Pull request merged**